package.json

```diff
@@ -23,6 +23,8 @@
     "lint": "eslint . bin/ospec"
   },
   "devDependencies": {
-    "eslint": "^6.8.0"
+    "compose-regexp": "0.4.0",
+    "eslint": "^6.8.0",
+    "ospec": "4.0.1"
```

Comment:
Let's just use the dev version, what we use for everything else - no need to recursively test it, and it's easier to maintain later.